main/ZgatewayRTL_433.ino

```diff
@@ -0,0 +1,135 @@
+/*  
+  OpenMQTTGateway  - ESP8266 or Arduino program for home automation 
+
+   Act as a wifi or ethernet gateway between your 433mhz/infrared IR signal  and a MQTT broker 
+   Send and receiving command by MQTT
+ 
+  This gateway enables to:
+ - receive MQTT data from a topic and send RF 433Mhz signal corresponding to the received MQTT data
+ - publish MQTT data to a different topic related to received 433Mhz signal
+ - leverage the rtl_433 device decoders on a ESP32 device
+
+    Copyright: (c)Florian ROBERT
+  
+    This file is part of OpenMQTTGateway.
+    
+    OpenMQTTGateway is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    OpenMQTTGateway is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include "User_config.h"
+
+#ifdef ZgatewayRTL_433
+
+#  include <rtl_433_ESP.h>
+
+#  define CC1101_FREQUENCY 433.92
+#  define JSON_MSG_BUFFER  512
+#  define ONBOARD_LED      2
+
+char messageBuffer[JSON_MSG_BUFFER];
+
+rtl_433_ESP rtl_433(-1); // use -1 to disable transmitter
+
+#  include <ELECHOUSE_CC1101_SRC_DRV.h>
+
+void rtl_433_Callback(char* message) {
+  DynamicJsonBuffer jsonBuffer2(JSON_MSG_BUFFER);
+  JsonObject& RFrtl_433_ESPdata = jsonBuffer2.parseObject(message);
+
+  pub(subjectRTL_433toMQTT, RFrtl_433_ESPdata);
+#  ifdef MEMORY_DEBUG
+  Log.trace(F("Post rtl_433_Callback: %d" CR), ESP.getFreeHeap());
+#  endif
+}
+
+void rtl_433setup() {
```

Comment:
Could you change it to setupRTL_433 please so as to follow the other namings for setup

Reply:
Done